components/scanners/zaproxy/internal/transformer/transformer.go

```diff
@@ -124,7 +124,10 @@ func (g *zapTransformer) Transform(ctx context.Context) ([]*ocsf.VulnerabilityFi
 		}
 		return nil, errors.Errorf("failed to read raw output file '%s': %w", g.rawOutFilePath, err)
 	}
-
+	if len(b) == 0 {
```

Comment:
please add empty lines around this block, same comment goes for most of the additions in this PR